ArduCopter/crash_check.cpp

```diff
@@ -169,12 +174,6 @@ void Copter::thrust_loss_check()
         // enable thrust loss handling
         motors->set_thrust_boost(true);
         // the motors library disables this when it is no longer needed to achieve the commanded output
-
-#if AP_GRIPPER_ENABLED
-        if (copter.option_is_enabled(FlightOption::RELEASE_GRIPPER_ON_THRUST_LOSS)) {
-            gripper.release();
-        }
-#endif
```

Comment:
We can't just remove this. The would need to be a arming check if the option is set and the feature is gone.